### PR TITLE
spell check

### PR DESCRIPTION
**Pull request by rscohn2**

### CODE REVIEW

**Comment by rscohn2** [comment]
Please check my spelling corrections and remember that there are changes to backport.

I will not wait for everyone before merging, but will respond to mistakes.

File: source/elements/oneMKL/source/domains/dft/config_params/strides.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ INPUT_STRIDES and OUTPUT_STRIDES
...
 :math:`X(k_1, k_2, \dots, k_d ) =` the :math:`+ s_0 + k_1*s_1 + k_2*s_2 + \dots + k_d*s_d`-th element of the container (``sycl::buffer`` or USM pointer) provided to the compute function,
-where :math:`s_0` is the displacement and :math:`s_1 , \dots, s_d` are generalized strides. The configuration parameters ``config_param::INPUT_STRIDES`` and ``config_param::OUTPUT_STRIDES`` enable you to get and set these values. The configuration value is a :math:`d+1`` lengthed ``std::vector<std::int64_t>`` of values :math:`(s_0, s_1, \dots, s_d )`.
+where :math:`s_0` is the displacement and :math:`s_1 , \dots, s_d` are generalized strides. The configuration parameters ``config_param::INPUT_STRIDES`` and ``config_param::OUTPUT_STRIDES`` enable you to get and set these values. The configuration value is a :math:`d+1`` with length ``std::vector<std::int64_t>`` of values :math:`(s_0, s_1, \dots, s_d )`.
```
Comment (timmiesmith) [inline source/elements/oneMKL/source/domains/dft/config_params/strides.rst:8]: "with length" should be "length".  The full phrase is "a d+1 length vector"

File: source/elements/oneTBB/source/thread_local_storage/flattened2d_cls.rst [hunk 2]
```diff
@@ -3,7 +3,7 @@ flattened2d
 ===========
 **[tls.flattened2d]**
 
-The class template ``flattened2d`` is an adaptor that provides a flattened view of a container of containers.
+The class template ``flattened2d`` is an adapter that provides a flattened view of a container of containers.
```
Comment (timmiesmith) [inline source/elements/oneTBB/source/thread_local_storage/flattened2d_cls.rst:6]: The C++ standard uses "adaptor"

**Comment by emfomenk** [comment]
Looks good wrt oneDNN. Thanks!

File: source/elements/oneMKL/source/domains/dft/config_params/strides.rst (hunk 1 above)
Comment (rscohn2) [reply source/elements/oneMKL/source/domains/dft/config_params/strides.rst]: changed

File: source/elements/oneTBB/source/thread_local_storage/flattened2d_cls.rst (hunk 2 above)
Comment (rscohn2) [reply source/elements/oneTBB/source/thread_local_storage/flattened2d_cls.rst]: changed

**Pull request merged**